WebServicesTesting/Jsons/UsersJson.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+
+namespace WebServicesTesting.Jsons
```

Comment:
It is a bad practice when all classes are contained in one place. Please separate them into different files